presto-docs/src/main/sphinx/release/release-0.207.rst

```diff
@@ -0,0 +1,71 @@
+=============
+Release 0.207
+=============
+
+General Changes
+---------------
+
+* Fix a planning issue for queries where correlated references were used in ``VALUES``.
+* Remove support for legacy ``JOIN ... USING`` behavior.
+* Change behavior for unnesting an array of ``row`` type to produce multiple columns.
+* Deprecate the ``reorder_joins`` session property and the ``reorder-joins`` configuration property.
+  Replace with the ``join_reordering_strategy`` session property and the
+  ``optimizer.join-reordering-strategy`` configuration propery. Setting this new property to ``NONE``
+  maintains the order of the joins as written and is equivalent to ``reorder-joins=false``.
+  ``ELIMINATE_CROSS_JOINS`` will eliminate any unnecessary cross joins from the plan and is
+  equivalent to ``reorder-joins=true``. Setting it to ``COST_BASED`` will use the new cost-based
+  optimizer to select the best join order. To simplify migration, setting the ``reorder_joins``
+  session property overrides the new session and configuration properties.
+* Deprecate the ``distributed_joins`` session property and the ``distributed-joins-enabled`` configuration
+  property. Replace with the ``join_distribution_type`` session property and ``join-distribution-type``
+  configuration property. Setting this new property to ``REPARTITIONED`` turns on hash partitioned
+  joins and is equivalent to ``distributed-joins-enabled=true``. Setting it to ``BROADCAST`` changes
+  the join strategy to broadcast and is equivalent to ``distributed-joins-enabled=false``. Setting it to
+  ``AUTOMATIC`` causes the engine to choose the join strategy based on the estimated cost of each join.
+  If no statistics are available, a hash-partitioning strategy is used. To simplify migration, setting
+  the ``distributed_joins`` session property overrides the new session and configuration properties.
+* Add optimizer to reorder inner joins and choose the join type based on cost when statistics are available.
+* Add support for column properties.
+* Add ``optimizer.max-reordered-joins`` configuration property to set the maximum number of joins that
+  can be reordered at once using cost-based join reordering.
+* Add support for ``char`` type to :func:`approx_distinct`.
+
+Security Changes
+----------------
+
+* Change server to fail on startup when configuration for file based system access control is invalid.
```

Comment:
Remove here

Reply:
It's both system and connector file access control.